changes

cordoval opened the pull request.

theodorDiaconu committed:
changes

jcroll committed:
Fixed doctrine mapping error

cordoval committed:
put body of cart into order entity

iampersistent commented on lib/Vespolina/Entity/Order/Item.php:
```diff
@@ -41,6 +43,25 @@ public function getId()
+        return $this->order;
+    }
+
+    /**
```
These are handled by setParent() and getParent()

Commit:
add pricing recurring element

theodorDiaconu committed:
added base partner

theodorDiaconu committed:
modified back to parent

theodorDiaconu committed:
removed duplicate

theodorDiaconu committed:
product

theodorDiaconu committed:
added ids

cordoval committed:
Merge pull request #6 from immersivelabs/feature/billing

add pricing recurring element

cordoval committed:
remove some line breaks

cordoval committed:
update partner interface to use the new interface

cordoval committed:
remove some line breaks

cordoval committed:
fix relations and add form types

cordoval committed:
wip

Commit:
updates to pricing

cordoval committed:
fix interface for pricing element

cordoval committed:
add arg to process on pricing element interface

cordoval committed:
switch setPrice to setPriceSet

cordoval committed:
alter getPriceSet to getPricingSet

theodorDiaconu committed:
pricing sets updates

theodorDiaconu committed:
base order updates

cordoval committed:
patch

jmgtan committed:
Added the attributes for the addresses collection in form type

theodorDiaconu committed:
updated pricing

jcroll committed:
Added constructor for one-to-many association

Commit:
addition information for the BillingAgreement

cordoval committed:
make details from partner fluent

cordoval committed:
add shortname property

jcroll committed:
Tell whether partner is paying by credit card

jcroll committed:
Added radio for isPayByCreditCard

theodorDiaconu committed:
added payment profile and linked partner to it

theodorDiaconu committed:
added pay profile on construct

cordoval committed:
fix docblock on OrderInterface

jcroll committed:
fluid interface

theodorDiaconu committed:
modified attributes for product

theodorDiaconu committed:
attribute interface back, and fixed item and orders

Commit:
additions to BillingAgreement

jcroll committed:
Trying to create inheritance

More work
Working now
Moved child classes to sub directory
More updates
More work on payment profile
New partner property paymentProfileType

Commit:
add initial state of some properties for BillingAgreement

Commit:
add BillingRequest class

cordoval committed:
add missing use statement and add slug fluency

theodorDiaconu committed:
events update

cordoval committed:
Update lib/Vespolina/Entity/Order/OrderEvents.php

theodorDiaconu committed:
fixed getParent @return

cordoval committed:
Merge pull request #8 from immersivelabs/feature/inheritance

[RFC] PaymentProfile/inheritance

jcroll committed:
Moved address to parent class

theodorDiaconu committed:
fixed entities to work with the new mapping

cordoval committed:
wip

cordoval committed:
fix couple of docblocks

jcroll committed:
More address properties

iampersistent commented:
manually merged
